ci-operator/config/openshift/cluster-openshift-apiserver-operator/openshift-cluster-openshift-apiserver-operator-master.yaml

```diff
@@ -37,14 +37,23 @@ tests:
 - as: verify
   commands: |
     go get -u golang.org/x/tools/imports
-    make verify -k
+    make verify -k --warn-undefined-variables
   container:
     from: bin
 
+- as: verify-deps
+  commands: |
+    HOME=$( mktemp -d )
+    go get -u github.com/Masterminds/glide
+    make verify-deps -k --warn-undefined-variables
+  container:
+    from: src
+
 - as: unit
   commands: |
     go get -u github.com/openshift/release/tools/gotest2junit
-    make test-unit JUNITFILE=/tmp/artifacts/unit_report.xml
+    mkdir /tmp/artifacts
```

Comment:
the current version semes to be not creating the dir anymore, adding the mkdir back